pysat/_instrument.py

```diff
@@ -175,11 +175,13 @@ def __init__(self, platform=None, name=None, tag=None, inst_id=None,
                  manual_org=None, directory_format=None, file_format=None,
                  temporary_file_list=False, strict_time_flag=True,
                  ignore_empty_files=False,
-                 units_label='units', name_label='long_name',
-                 notes_label='notes', desc_label='desc',
-                 plot_label='label', axis_label='axis', scale_label='scale',
-                 min_label='value_min', max_label='value_max',
-                 fill_label='fill', *arg, **kwargs):
+                 labels={'units': ('units', str), 'name': ('long_name', str),
+                         'notes': ('notes', str), 'desc': ('desc', str),
+                         'plot': ('plot_label', str), 'axis': ('axis', str),
```

Comment:
I don't think 'label' and 'plot_label' is consistent everywhere for `'plot`. I'm getting 'label' and 'plot_label' for your latest tests.

```
pysat/tests/test_meta.py::TestBasics::test_inst_data_assign_meta[labels2-vals2]

meta 
 units                                     
long_name       test_inst_data_assign_meta
notes                                     
desc                                      
plot_label      test_inst_data_assign_meta
axis            test_inst_data_assign_meta
scale                               linear
value_min                              NaN
value_max                              NaN
fill                                  -999
custom                                 NaN
Bin_Location                           NaN
label                                  NaN
children                              None
Name: test_inst_data_assign_meta, dtype: object

```

Reply:
I also had to change a few lines in test_meta.py:

```
        self.default_name = ['long_name', 'axis', 'plot_label']

```
under `TestBasicsImmutable` and `TestBasics`.

Reply:
It looks like `plot_label` is assigned by default, but `label` is being specifically asked for by that test.  So it's testing the setting of a custom MetaLabel in this instance.